scm/branching-model/feature-toggling.md

```diff
@@ -10,23 +12,23 @@ Workflow, Process Builders and Visual Workflows \(branching condition that check
 
 ### App Logic - Code <a id="user-content-app-logic---code"></a>
 
-* Using a Trigger Framework which allows for deactivation of any given trigger handler via a metadata type
-* Use a framework to read values from a custom metadata object to determine whether to execute code
+* Use a Trigger Framework which allows for deactivation of any given trigger handler via a Custom Metadata Type or Custom Permission. [Apex Trigger Actions](https://github.com/mitchspano/apex-trigger-actions-framework) and [Nebula Triggers](https://bitbucket.org/nebulaconsulting/nebula-core/src/master/) are examples of open source framerworks which provide this capability
```

Comment:
I am not sure if the ability to enable or disable a trigger handler is desirable feature. Probably worth discussing the use cases before promoting this.
Also typo in the word "frameworks"

Reply:
Thats probably me..as part of the original content, will wait for your both decision and change accordingly

Reply:
My view would be this can be a useful and straightforward way to toggle for some circumstances - for example a trigger handler which calls a service to make a REST request to a new external system and update the record with the result can be inactive until the system is up.

Worth mentioning the meaning of "handler" under each of these frameworks would be a discrete apex class doing one aspect of trigger processing, so disabling one class still allows other logic from the trigger to run